pbsmetrics/go_metrics.go

```diff
@@ -234,6 +244,11 @@ func makeBlankAdapterMetrics() *AdapterMetrics {
 		BidsReceivedMeter: blankMeter,
 		PanicMeter:        blankMeter,
 		MarkupMetrics:     makeBlankBidMarkupMetrics(),
+		ConnError:         metrics.NilCounter{},
+		ConnCreated:       metrics.NilCounter{},
+		ConnReused:        metrics.NilCounter{},
+		ConnIdleTime:      &metrics.NilTimer{},
```

Comment:
I thought it'd be useful to know which connections are more efficient. I'm not sure if I'm going for a "nice to have" here instead of a "must have" or we "definitely need" this. Do think we should discard them?